acceptance-tests.md

```diff
@@ -66,10 +79,12 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Set and modify quotas for projects**
     * _As an administrator of the cluster, I should be able to set and modify compute, storage and object counts quotas for any project._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
```

Comment:
This is not clear about how changes to allocations are made.  They are made in ColdFront and have to be passed to OpenShift.  I don't believe they will be made in OpenShift directly, because the two would get out of sync.

Reply:
Description was revised to make this clear.